app/models/comment.rb

```diff
@@ -230,6 +230,8 @@ def self.add_answer_comment(mail, answer_id, user)
                   gmail_parsed_mail mail_doc
                 elsif domain == "yahoo"
                   yahoo_parsed_mail mail_doc
+		elsif domain == "outlook"
```

Comment:
Hmm, is this getting mixed up with commits from another PR? Just wondering!!!

Reply:
Yes it got mixed up... How can I resolve this? 😢

Reply:
I'm not so familiar with git and github.. Please help